src/openvic-simulation/misc/Modifier.hpp

```diff
@@ -105,10 +112,35 @@ namespace OpenVic {
 	struct ModifierInstance {
 
 	private:
-		Modifier const& PROPERTY(modifier);
+		Modifier const* PROPERTY(modifier); // We can assume this is never null
 		Date PROPERTY(expiry_date);
 
-		ModifierInstance(Modifier const& modifier, Date expiry_date);
+	public:
+		ModifierInstance(Modifier const& new_modifier, Date new_expiry_date);
+	};
+
+	struct ModifierSum {
```

Comment:
Modifier is pretty big. It might be better to move the different structs to different files.
Modifiers can be moved from /misc to /modfiers then

Reply:
Yeah I was thinking that too, I'll do that in an extra commit on this PR